add docs for vdc deployment (#2830)
threefoldtech/js-sdk: docs/vdc/vdc_deployer.md

# VDC Deployer

> This manual will go through setting up an environment for vdc deployments and quick walk through the sals

Reviewer: ```suggestion
This manual will go through setting up an environment for vdc deployments and a quick walk through the sals.
```

## Requirments

- JS-SDK
- Machine with a domain
- Stellar wallets
  - vdc_init: Used to fund the initialization of vdcs to make sure it's working to save users money, also it's used to fund the plan differences. (Required asset: TFTs)
  - grace_period: Used to fund the 2 weeks grace period. (Required asset: TFTs)

>   - activation_wallet: Used to activate vdc wallets (provisioning_wallet & prepaid_wallet) in case of threefold service failed to activate. (Required asset: XLMs)

Reviewer 2: ```suggestion
  - activation_wallet: Used to activate vdc wallets (provisioning_wallet & prepaid_wallet). (Required asset: XLMs)
```

>   - vdc_init: Used to fund the initialization of vdcs to make sure it's working to save users money, also it's used to fund the plan differences. (Required asset: TFTs)
>   - grace_period: Used to fund the 2 weeks grace period. (Required asset: TFTs)
>   - activation_wallet: Used to activate vdc wallets (provisioning_wallet & prepaid_wallet) in case of threefold service failed to activate. (Required asset: XLMs)

Reviewer: ```suggestion
  - **vdc_init**: Used to fund the initialization of vdcs to make sure it's working to save users money, also it's used to fund the plan differences. (Required asset: TFTs)
  - **grace_period**: Used to fund the 2 weeks grace period. (Required asset: TFTs)
  - **activation_wallet**: Used to activate vdc wallets (provisioning_wallet & prepaid_wallet) in case of threefold service failed to activate. (Required asset: XLMs)
```

> ## Setup

Reviewer 3: Can we add the s3 backup server requirements as well?

- Install [JS-SDK](https://threefoldtech.github.io/js-sdk/wiki/#/./quick_start)

> - Create/Import the required wallets as in requirments through `jsng` shell

Reviewer: ```suggestion
- Create/Import the required wallets mentioned in the previous requirements through `jsng` shell
```
